IPIP/0351-gateway-ipns-record-response-format.md

```diff
@@ -0,0 +1,77 @@
+# IPIP 0351: IPNS Signed Records Response Format on HTTP Gateways
+
+- Start Date: 2022-11-29
+- Related Issues:
+  - [ipfs/specs/issues/320](https://github.com/ipfs/specs/issues/320)
+  - [ipfs/specs/pull/351](https://github.com/ipfs/specs/pull/351)
+  - [ipfs/kubo/pull/9399](https://github.com/ipfs/kubo/pull/9399)
+
+## Summary
+
+Add IPNS Signed Records response format to the [HTTP Gateway](../http-gateways/).
+
+## Motivation
+
+Currently, the gateway allows for trustless retrieval of data under the `/ipfs`
+namespace, bu fetching the data as a CAR, or Block, and then verifying it locally.
+This is especially important for light IPFS clients, so that they can retrieve
+data from other gateways without delegating any of the trust to them. Unfortunately,
+this is not possible under the `/ipns` namespace.
+
+In contrary to DNSLink, IPNS provides cryptographically-verifiable records that
+can be verified by the client. This means that, if a gateway is able to provide
+the IPNS signed record to an HTTP client, trustless retrieval would also be available
+under the `/ipns` namespace.
+
+In this IPIP, we propose adding [IPNS signed records][ipld-record] as a response
+format to the gateway under the `/ipns` namespace, allowing for trustless
+retrieval of IPNS records.
+
+## Detailed design
+
+The solution is to allow the Gateway to provide an IPNS signed record by
```

Comment:
Should Gateway be capitalized as it is here? (It's lowercase in other places on this doc)